modules/my_preferences/README.md

```diff
@@ -20,9 +20,5 @@ A user is always able to access the My Preferences page.
 
 ## Configurations
 
-Some settings related to email notifications affect this module.
-
-## Interactions with LORIS
-
-- Modifications performed on the My Preferences page will be reflected on 
-the user account page when viewing the edited account.
+This module allows a user to modify whether or not they are notified by events in other modules,
```

Comment:
@johnsaigle I thought this section is for configurations that affect this module, not configurations that are affected by it.

not sure though

Reply:
Yes.

Reply:
Thanks for clarifying. As far as I know there aren't any.